### ISSUE TITLE
Change "shall" to "must" for consistency with other rules

### ISSUE THREAD

**Issue by evandroAlfieri** [buildingSMART/ifc-gherkin-rules#20]
https://github.com/buildingSMART/ifc-gherkin-rules/blob/462daf5ccf032fad7e0d5245825aff4f3604736b/features/SPS001_Basic-spatial-structure-for-buildings.feature#L10

**Comment by pjanck** [comment]
IMHO, I'd rather use shall everywhere else. See Table 3 and Section 7.6 [here](https://www.iso.org/sites/directives/current/part2/index.xhtml).

**Comment by evandroAlfieri** [comment]
Consistency of rules for the bSI Validation Service is important. As bSI responsible for such service, I took this seriously and I've conducted an analysis on the matter (I never thought it would have been this fun, keep reading). The conclusion I came to **must** be followed for every rule developed for the bSI Validation Service. *Go to the end of this long answer to read the conclusion*.

On March 22nd 1995, the US Supreme Court heard the case of [De Martinez v. Lamagno](https://www.law.cornell.edu/supct/html/94-167.ZS.html). Making a very long story short, the case came down to one word: "shall".

Extract from the sentence:
> [...] upon certification by the attorney general, any civil action or proceeding **shall** be deemed in action against the United State and the United States **shall** be substituted as the party defendant.

The Court decided that the word **shall** has a legal definition of **may**, not **must**. Meaning that the Government was allowed to substitute the U.S. for the defendant, but was not required to. This meant that, from that day on, in the U.S. the word "shall" would now be interpreted as "may".

The US government has been very clear about it. Here is what the [U.S. government website](https://www.plainlanguage.gov/guidelines/conversational/shall-and-must/) says:
> Use **must** not **shall** to impose requirements. “Shall” is ambiguous, and rarely occurs in everyday conversation. The legal community is moving to a strong preference for “must” as the clearest way to express a requirement or obligation.

Now, this was happening in America. Shortly after, in Canada lawyers began to talk about that. In 2011, Australia and UK both changed shall to must. And then Canada did.

From the official [Canadian government website](https://www.justice.gc.ca/eng/rp-pr/csj-sjc/legis-redact/legistics/p1p8.html)
> The auxiliary “shall” is not to be used, because of its legalistic tone, its rarity in Canadian English outside legal documents and the multiple meanings that have been ascribed to it in legislative texts.[1]
[...]
> There are several ways to draft prohibitions: “X must not …”, “A person must not …”, “It is prohibited to …”
[…]
> “May not” and “no person may” **must** not be used to create prohibitions.  “May” in the positive has quite a different meaning from “shall” or “must”; however, the meaning of these words in common usage is more or less the same when they are made negative: that something is forbidden.

With regards to the ISO/IEC directives for drafting an ISO/IEC standard, these are, as the title suggests, principles for drafting **ISO documents**, not for the definition of requirements in legal terms, or in general.

In particular, the distinction between "must" and "shall", as expressed by the note in Table 7, is to avoid mixing *provisions" and *external constraints*. (*external constraint*: constraint or obligation on the user of the document that is not stated as a *provision* of the document; *provision*; expression in the content of a normative document that takes the form of a statement, an instruction, a recommendation or a requirement).

Such in-document versus outside-document distinction does not apply to the rules of the Validation Service, being the rules explicit constraints and obligations on the use of the IFC standard, not recommendations nor requirements that may or may not be fulfilled).

**CONCLUSION**
After listening to all the evidence 😄 ...it appears that in many relevant cases, the English legislation avoids the use of “will” or “shall” in favour of “must”, where “must” always suggests an absolute obligation. And the word “shall”, as long as it is used, it means may, and not must. 

❗ For all these reasons, every rule of the Validation Service prescribing an obligation **must** 😉 use the word “must”.
